Submitted By:            Pierre Labastie
Date:                    2011-12-27
Initial Package Version: 1.4.5 (taken from BLFS patch for httpd-2.2.0)
Upstream Status:         Not submitted (jhalfs specific)
Origin:                  jhalfs
Description:             Modify the layout for installing Apache Portable Runtime

diff -Naur apr-1.4.5-orig/config.layout apr-1.4.5/config.layout
--- apr-1.4.5-orig/config.layout	2004-11-24 22:51:51.000000000 +0000
+++ apr-1.4.5/config.layout		2005-12-14 21:06:37.000000000 +0000
@@ -11,18 +11,18 @@
 
 #   Classical APR path layout designed for parallel installs.
 <Layout apr>
-    prefix:        /usr/local/apr
+    prefix:        /usr
     exec_prefix:   ${prefix}
     bindir:        ${exec_prefix}/bin
-    sbindir:       ${exec_prefix}/bin
+    sbindir:       ${exec_prefix}/sbin
     libdir:        ${exec_prefix}/lib
-    libexecdir:    ${exec_prefix}/modules
-    mandir:        ${prefix}/man
-    sysconfdir:    ${prefix}/conf
-    datadir:       ${prefix}
-    installbuilddir: ${datadir}/build-${APR_MAJOR_VERSION}
-    includedir:    ${prefix}/include/apr-${APR_MAJOR_VERSION}
-    localstatedir: ${prefix}
+    libexecdir:    ${exec_prefix}/lib/apache
+    mandir:        ${prefix}/share/man
+    sysconfdir:    /etc/apache
+    datadir:       /srv/www
+    installbuilddir: ${libexecdir}/build
+    includedir:    ${prefix}/include/apache
+    localstatedir: ${datadir}
     libsuffix:     -${APR_MAJOR_VERSION}
 </Layout>
